README.md

```diff
@@ -1,5 +1,8 @@
 # Frequenz channels
 
+[<img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/frequenz-floss/frequenz-channels-python/frequenz-channels-python?label=docs">](https://frequenz-floss.github.io/frequenz-channels-python/)
```

Comment:
If we had a separate workflow for CI and release,  we could use the release workflow for this.  We can of course a static badge as well.  Let me make one just to see.

Reply:
How does this look: https://github.com/shsms/frequenz-channels-python/blob/readme-badges/README.md

Reply:
Yeah, I tried doing that, but I had to duplicate some steps, which wasn't ideal. There is probably a way to do it if we can use one workflow result in another. At the time I tried it briefly but it was complicated so I preferred to keep it simple for now. But I agree it would be ideal to have more independent workflows than one giant with everything.